.gitignore

```diff
@@ -32,3 +32,9 @@ examples/SLSQP.out
 
 # Log files
 *.log
+
+# User-specific files
```

Comment:
Are these relevant to keep or left over from local changes?

Reply:
Local changes, good catch! They are removed now.